docs/cudf/source/user_guide/10min.ipynb

```diff
@@ -2,20 +2,25 @@
  "cells": [
```

Comment:
<p>Can we edit the raw output here too? I think Bradley had a tool to clean up these paths.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12114/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='167'/>

Reply:
https://github.com/bdice/nb-strip-paths

Must be run on the local machine before committing — CI won’t know what paths are local. I filed a SWIPAT but I’m not sure if it was completed or not.

Reply:
Yes.